addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1,6 +1,9 @@
 odoo.define('web_editor.wysiwyg', function (require) {
 'use strict';
 
+const { ComponentWrapper } = require('web.OwlCompatibility');
+const { MediaDialogWrapper } = require('@web_editor/components/media_dialog/media_dialog');
+const { saveVideos, videoSpecificClasses } = require('@web_editor/components/media_dialog/video_selector');
```

Comment:
Hello ! Just reporting something that has been noticed by our new colleague. Was this line intended to import the specific classes defined there https://github.com/odoo/odoo/pull/89223/files#diff-7bd982ca47856298219e5e4e88e8ebe557f32f938f111b0b02c36e034c6ef8baR195 ? Because the names of the property are different and they're not even exported per se, they're available as static properties on the class. I can't find the `saveVideos` function either. I am very confused. Am I missing something or was there some refactoring that has been done during the development of this task and those imports were forgotten ?

Reply:
Seems like an error indeed :scream:  @younn-o converted the media dialog to OWL, maybe he can help or you can leave that to us entirely if you prefer. Although I see that before this merge, that part of the code was invoking a whole video dialog just to create a video node (https://github.com/odoo/odoo/commit/e2763e0f8ba5d2f7c520c31cf8c108233f0c7504). Should not some utils be made? Maybe that `saveVideos` was an attempt at doing that which was forgotten later on indeed.

@younn-o To maybe save you some time if you want to have a look:
- Edit mode
- Paste a youtube link in the DOM
- The powerbox opens, click on "Embed youtube video"
=> Crash

Reply:
Yes, it's definitely a mistake from my end, sorry for that! I did not adapt the Wysiwyg after changing to use static methods and properties for the VideoSelector. I think it should be `VideoSelector.createElements` instead of `saveVideos` and `VideoSelector.mediaSpecificClasses` instead of `videoSpecificClasses`, I will open a PR to fix that

Reply:
> I will open a PR to fix that

Thanks, seems straightforward indeed :+1: Maybe the `mediaSpecificClasses` stuff could be defined as a static member inside the class actually? I have no experience with the `static` keyword but I guess it should work? Then maybe just import VideoSelector entirely and use it that way (would be strange to extract static members of a Class with destructuring IMO, if that even works).

@dmo-odoo Side note about https://github.com/odoo/odoo/commit/e2763e0f8ba5d2f7c520c31cf8c108233f0c7504 -> I see the videowidget is not destroyed, maybe a slight cause of memory leak :wink: